scripts/mkcloud

```diff
@@ -15,6 +15,15 @@
 # 2. This tool relies on the script qa_crowbarsetup.sh
 # 3. Please 'export' environment variables according to your needs.
 
+if [[ "$cloud" ]]; then
```

Comment:
Why not move this line
`: ${cloud:=cloud}`
above this block and skip the check?

Btw: no quotes required.